e2e/app/page/notebook-page.ts

```diff
@@ -45,7 +45,7 @@ export default class NotebookPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     await waitForDocumentTitle(this.page, this.documentTitle);
-    await this.waitForKernelIdle();
+    await this.waitForKernelIdle(30000);
```

Comment:
wait longer for kernel status becomes ready. Notebook test failed in https://app.circleci.com/pipelines/github/all-of-us/workbench/5309/workflows/6ffd0b63-da62-477c-b54f-448bb6491880/jobs/106914/tests

stacktrace:
```
Failed test:  "Workspace-owner-Jupyter-notebook-action-tests/Copy-notebook-to-another-Workspace-when-CDR-versions-match"
Error: waitForKernelIdle encountered TimeoutError: waiting for selector "#kernel_indicator_icon.kernel_idle_icon" failed: timeout 10000ms exceeded
  console.error
    Notebook kernel is: Kernel Busy

      119 |       ]);
      120 |     } catch (e) {
    > 121 |       console.error(`Notebook kernel is: ${await this.kernelStatus()}`);
          |               ^
      122 |       throw new Error(`waitForKernelIdle encountered ${e}`);
      123 |     }
      124 |   }

      at NotebookPage.waitForKernelIdle (app/page/notebook-page.ts:121:15)
      at NotebookPage.isLoaded (app/page/notebook-page.ts:48:5)
      at NotebookPage.waitForLoad (app/page/authenticated-page.ts:35:5)
      at WorkspaceAnalysisPage.createNotebook (app/page/workspace-analysis-page.ts:77:5)
      at copyNotebookTest (tests/notebook/owner-copy-to-workspace.spec.ts:31:31)
      at Object.<anonymous> (tests/notebook/owner-copy-to-workspace.spec.ts:75:5)
```

Reply:
I am ok with this fix in this PR - do we know the upper limit where we want to fail here? Is it at 30s? Or would waiting for the browser default timeout be ok?

Reply:
30 seconds is a tryout. hopefully good enough. I don't want to start with a bigger timeout unless it's absolutely necessary. I would think 60 seconds is the upper limit.